src/applications/check-in/travel-claim/pages/complete/TravelClaimSuccessAlert.jsx

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
+import { format } from 'date-fns';
 import { getUniqueFacilies } from '../../../utils/appointment';
 import { formatList } from '../../../utils/formatters';
 
 const TravelClaimSuccessAlert = props => {
-  // @TODO refactor this once we have worked out where to derive facilities and times for claims from
-  const { appointments } = props;
-  const facilities = getUniqueFacilies(appointments);
-
+  const { claims } = props;
```

Comment:
If claims is undefined or null, this next part (the `if (claims.length...`) will blow up with a Typeerror

Reply:
oh good point. I'm going to set the prop to required anyway.